vortexasdk/api/vessel.py

```diff
@@ -53,10 +53,10 @@ class Vessel(Node):
     corporate_entities: Optional[List[VesselEntityCorporateEntity]] = None
     dead_weight: Optional[int] = None
     cubic_capacity: Optional[int] = None
-    to_bow: Optional[str] = None
-    to_stern: Optional[str] = None
-    to_port: Optional[str] = None
-    to_starboard: Optional[str] = None
+    to_bow: Optional[int] = None
```

Comment:
Were these incorrectly typed previously?

Reply:
So it seems. The older version of Pydantic converted them but the newer version is stricter.